series.go

```
@@ -0,0 +1,45 @@
+// Copyright 2014 Canonical Ltd.
+// Licensed under the LGPLv3, see LICENCE file for details.
+
+package names
+
+import (
+	"fmt"
+	"regexp"
+)
+
+// ValidSeries matches any valid series string.
+var validSeries = regexp.MustCompile("^" + SeriesSnippet + "$")
+
+// KnownSeries is a map of all the different series' that juju knows about.
+var KnownSeries = map[string]bool{
+	"bundle":      true,
+	"oneiric":     true,
+	"precise":     true,
+	"quantal":     true,
+	"raring":      true,
+	"saucy":       true,
+	"trusty":      true,
+	"utopic":      true,
+	"vivid":       true,
+	"win2012hvr2": true,
+	"win2012hv":   true,
+	"win2012r2":   true,
+	"win2012":     true,
+	"win7":        true,
+	"win8":        true,
+	"win81":       true,
+	"centos7":     true,
+}
+
+// VerifySeries checks to see if a given string is a valid series string and if
+// it is a known series.
+func VerifySeries(series string) (bool, error) {
```

I think we could just return an error here. The series is not valid if the err != nil.